### PR TITLE
Added missing codec config options

### PR DESCRIPTION
**Pull request by moser4035**

### CODE REVIEW

File: bitmovin/resources/enums/chroma_location.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class ChromaLocation(enum.Enum):
+    UNSPECIFIED = 'UNSPECIFIED'
+    LEFT = 'LEFT'
+    CENTER = 'CENTER'
+    TOPLEFT = 'TOPLEFT'
+    TOP = 'TOP'
+    BOTTOMLEFT = 'BOTTOMLEFT'
+    BOTTOM = 'BOTTOM'
+
+    @staticmethod
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/chroma_location.py]: do you actually need this default here?

File: bitmovin/resources/enums/color_primaries.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    FILM = 'FILM'
+    BT2020 = 'BT2020'
+    SMPTE428 = 'SMPTE428'
+    SMPTEST428_1 = 'SMPTEST428_1'
+    SMPTE431 = 'SMPTE431'
+    SMPTE432 = 'SMPTE432'
+    JEDEC_P22 = 'JEDEC_P22'
+
+    @staticmethod
+    def default():
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/color_primaries.py]: do you actually need this default here?

File: bitmovin/resources/enums/color_range.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+
+class ColorRange(enum.Enum):
+    UNSPECIFIED = 'UNSPECIFIED'
+    MPEG = 'MPEG'
+    JPEG = 'JPEG'
+
+    @staticmethod
+    def default():
+        return ColorRange.UNSPECIFIED
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/color_range.py]: do you actually need this default here?

File: bitmovin/resources/enums/color_space.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    SMPTE240M = 'SMPTE240M'
+    YCGCO = 'YCGCO'
+    YCOCG = 'YCOCG'
+    BT2020_NCL = 'BT2020_NCL'
+    BT2020_CL = 'BT2020_CL'
+    SMPTE2085 = 'SMPTE2085'
+
+    @staticmethod
+    def default():
+        return ColorSpace.UNSPECIFIED
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/color_space.py]: do you actually need this default here?

File: bitmovin/resources/enums/color_transfer.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    IEC61966_2_1 = 'IEC61966_2_1'
+    BT2020_10 = 'BT2020_10'
+    BT2020_12 = 'BT2020_12'
+    SMPTE2084 = 'SMPTE2084'
+    SMPTE428 = 'SMPTE428'
+    ARIB_STD_B67 = 'ARIB_STD_B67'
+
+    @staticmethod
+    def default():
+        return ColorTransfer.UNSPECIFIED
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/color_transfer.py]: do you actually need this default here?

File: bitmovin/resources/enums/input_color_range.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+
+class InputColorRange(enum.Enum):
+    UNSPECIFIED = 'UNSPECIFIED'
+    MPEG = 'MPEG'
+    JPEG = 'JPEG'
+
+    @staticmethod
+    def default():
+        return InputColorRange.UNSPECIFIED
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/input_color_range.py]: do you actually need this default here? please get rid of that unspecified here, i don't think we need this. how does that look at the API side?

File: bitmovin/resources/enums/input_color_space.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    SMPTE170M = 'SMPTE170M'
+    SMPTE240M = 'SMPTE240M'
+    YCGCO = 'YCGCO'
+    YCOCG = 'YCOCG'
+    BT2020_NCL = 'BT2020_NCL'
+    BT2020_CL = 'BT2020_CL'
+    SMPTE2085 = 'SMPTE2085'
+
+    @staticmethod
+    def default():
```
Comment (dominic-miglar) [inline bitmovin/resources/enums/input_color_space.py]: do you actually need this default here?

File: bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py [hunk 8]
```diff
@@ -59,6 +63,23 @@ def __init__(self, name, bitrate, rate, profile, id_=None, description=None, cus
...
+    @property
+    def colorConfig(self):
+        return self._colorConfig
+
+    @colorConfig.setter
+    def colorConfig(self, new_color_config):
+        if new_color_config is None:
+            self._colorConfig = None
+        elif isinstance(new_color_config, ColorConfig):
+            self._colorConfig = new_color_config
```
Comment (dominic-miglar) [inline bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py]: elsewhere we're always setting the .value of the enum here?

File: bitmovin/resources/enums/input_color_range.py (hunk 6 above)
Comment (moser4035) [reply bitmovin/resources/enums/input_color_range.py]: This UNSPECIFIED is an value that is defined in the API spec

File: bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py (hunk 8 above)
Comment (moser4035) [reply bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py]: What enum do you mean? colorConfig is an Object

File: tests/bitmovin/services/codecconfigurations/h265_codec_configuration_tests.py [hunk 9]
```diff
@@ -1,8 +1,10 @@
 import unittest
 import json
 from bitmovin import Bitmovin, Response, H265CodecConfiguration, H265Profile, H265Level, BAdapt, MaxCTUSize, \
-    TUIntraDepth, TUInterDepth, MotionSearch
+    TUIntraDepth, TUInterDepth, MotionSearch, ChromaLocation, ColorSpace, ColorPrimaries, ColorRange, ColorTransfer, \
+    InputColorSpace, InputColorRange
```
Comment (dominic-miglar) [inline tests/bitmovin/services/codecconfigurations/h265_codec_configuration_tests.py]: why don't you import ColorConfig also from that line here?

File: tests/bitmovin/services/codecconfigurations/h264_codec_configuration_tests.py [hunk 10]
```diff
@@ -1,12 +1,14 @@
 import unittest
-from bitmovin import Bitmovin, Response, H264CodecConfiguration, H264Profile, MVPredictionMode, H264Level, BAdapt
+from bitmovin import Bitmovin, Response, H264CodecConfiguration, H264Profile, MVPredictionMode, H264Level, BAdapt, \
```
Comment (dominic-miglar) [inline tests/bitmovin/services/codecconfigurations/h264_codec_configuration_tests.py]: why don't you import colorconfig also from here?

**Pull request merged**